src/components/DropCliff.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from '../../styled-system/css'
+import Path from '../@types/Path'
+import { isTouch } from '../browser'
+import { HOME_PATH } from '../constants'
+import head from '../util/head'
+import unroot from '../util/unroot'
+import DropEnd from './DropEnd'
+
+/** Renders a drop target for dropping thoughts after a cliff. */
```

Comment:
This component actually renders multiple drop targets: one for every level of the cliff.